Aztec/Classes/TextKit/TextView.swift

```diff
@@ -339,6 +339,12 @@ open class TextView: UITextView {
         }
     }
 
+    override open var textAlignment: NSTextAlignment {
+        didSet {
+            recalculateTypingAttributes()
```

Comment:
I sugest we had a check like this:

```
if oldValue != textAlignment {
  recalculateTypingAttributes()
}
```

Reply:
Good catch @SergioEstevao ! I made the update and it works great.

I was curious how large of a difference this would make, so I did some logging and I was surprised that the alignment prop actually isn't getting passed in as often as I expected. For example, I expected that changing the formatting of text (bold, italic, etc.) in an aligned paragraph would resend the alignment attribute to the native view, but it didn't. This is still definitely a good change because there were certainly some unnecessary calls to `recalculateTypingAttributes()` with my original implementation. I was just surprised that it wasn't an even more important fix.